### DISCUSSION TITLE
‼ ERROR 1@77 You need to set 'CTID' variable.

### DISCUSSION THREAD

**Discussion by andersot72** [tteck/Proxmox#434] [Help Needed] [answered]
New to Proxmox LXC Containers was looking to understand why I am getting this error, regardless of any software I try to deploy via the bash scripts here.  Thanks in advance

**Comment by tteck** [comment, marked as answer]
Are you using default settings?

**Reply by andersot72** [reply]
I am an idiot I figured out what I was doing wrong.  Apologies for wasting anyones time
